feat(curriculum): adding superhero application form (React forms) workshop (#59543)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-react-forms/67ebbf730d448c1d98d8dffc.md

---
id: 67ebbf730d448c1d98d8dffc
title: Step 3
challengeType: 0
dashedName: step-3
---

# --description--

Just after the `"Please complete all fields"` paragraph, create a `form` element. Inside it, create a `div` with a `className` of `section`.

# --hints--

You should create a `form` element.

```js
assert.exists(document.querySelector("form"));
```

You should create a `div` element inside your `form` element.

```js
assert.exists(document.querySelector("form > div"));
```

> Your `div` element should have `section` as its `className`.

Reviewer: ```suggestion
Your `div` element should have a `className` of `section`.
```
